functions/Invoke-DbaAdvancedUpdate.ps1

```diff
@@ -40,6 +40,9 @@ Function Invoke-DbaAdvancedUpdate {
         This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
         Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
 
+    .PARAMETER ExtractPath
+        Lets you specify a location to extract the update file to on the system requiring the update. e.g. C:\temp
+
     .EXAMPLE
     PS C:\> Invoke-DbaAdvancedUpdate -ComputerName SQL1 -Action $actions
```

Comment:
An example should be added for docs to draw attention to the usage of the parameter added.

Reply:
got it

Reply:
because Invoke-DbaAdvancedUpdate is called by Update-Dbainstace I wasn't sure how valid an example would be?

Reply:
This is not an internal only function, it is public and usable by any user. So an example is needed.